### PR TITLE
fix some credo issues

### PR DESCRIPTION
**Pull request by davydog187**
There are still some consistency issues, but we can fix those with the formatter (which I will do in another PR if thats cool).

### CODE REVIEW

File: lib/floki/raw_html.ex [hunk 1]
```diff
@@ -1,4 +1,8 @@
 defmodule Floki.RawHTML do
+  @moduledoc"""
```
Comment (philss) [inline lib/floki/raw_html.ex]: I prefer not to document modules that are "private" for now. I think the the docs from the "public" function are enough: https://hexdocs.pm/floki/Floki.html#raw_html/1
Could you please check it to `@moduledoc false`?

**Comment by davydog187** [comment]
@philss went ahead and ran the formatter as part of this PR, also addressed the remaining consistency issues.

**Comment by davydog187** [comment]
It would be helpful if ebert was configured to run `mix format --check-fromatted` and `mix credo`

**Comment by philss** [comment]
> It would be helpful if ebert was configured to run mix format --check-fromatted and mix credo

I think I can add this to the build, and fail if it is not following the format. Not sure if I can integrate with credo. If you know how, please go ahead with this too.

Thank you! 💜

**Pull request merged**